raster/r.walk/r.walk.html

```diff
@@ -152,6 +152,13 @@ <h2>EXAMPLES</h2>
 r.contour walkcost output=walkcost step=1000
 </pre></div>
 
+<div align="center" style="margin: 10px">
+<a href="r_walk.png">
+<img src="r_walk.png" width="600" height="600" alt="r.walk example" border="0">
+</a><br>
+<i>Figure: Walkshed over a cost surface derived from landcover</i>
```

Comment:
This is minor, but the cost surface is derived also based on elevation, so it may be worth clarifying that in the caption.